cmd/minishift/cmd/registration/util.go

```diff
@@ -48,7 +48,7 @@ func RegisterHost(libMachineClient *libmachine.Client) {
 	}
 
 	if wasSuccessful, err := cluster.Register(libMachineClient); err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Error to register VM: %v", err))
+		atexit.ExitWithMessage(1, "   Error in registering VM")
```

Comment:
@anjannath Why not add this error message on debug logs? cc @LalatenduMohanty @gbraad

Reply:
Yes, Adding the `err` in higher log level i.e. V2 would be a good idea as it might have information that can help us debug.

Reply:
adding an error in a debug or still dump on the console is still risky, unless you do some stripping of credentials when they are returned!

else, I am NOT in favor of printing an error that comes from this step

Reply:
is there a way to redact the message? As what I believe the password and username/email comes from the command WE provide... so this is a pattern that we can match and modify.

Reply:
have a look at `github.com/minishift/minishift/pkg/minishift/cluster`.`Register()`

Reply:
@gbraad We discussed on IRC and removing the password was suggested by @praveenkumar too and we all agreed to it.

Reply:
I provide the context of where I expect this change to happen.

i also hope he will look at the hsitory of these files, as it will help him with a nice refactor from which he can learn something.